server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -130,6 +132,77 @@ pub(crate) struct RequisitionTransferTester {
     response_requisition: Option<RequisitionRow>,
 }
 
+/// Deleted requisitions stuck forever
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
Test fails and freezes when using tokio::test (which is usually single threaded test)